### PR TITLE
[SDK-554] Adding ReceiptID to ActivityDetails

### PR DESCRIPTION
**Pull request by echarrod**
- Added ReceiptID to ActivityDetails
- Updated ActivityDetails items to be privately set
- Corrected package locations for 45Example project
- Changed CoreExample project to use inline QR code

### CODE REVIEW

File: src/Yoti.Auth/ActivityDetail.cs [hunk 1]
```diff
@@ -12,20 +12,42 @@ public enum ActivityOutcome { Success, ProfileNotFound, Failure, SharingFailure
...
         /// The <see cref="YotiProfile"/> returned by Yoti if the request was successful.
         /// </summary>
-        public YotiProfile Profile { get; set; }
+        public YotiProfile Profile { get; private set; }
 
         /// <summary>
         /// The outcome status of the request.
         /// </summary>
-        public ActivityOutcome Outcome { get; set; }
+        public ActivityOutcome Outcome { get; private set; }
```
Comment (bucky-boy) [inline src/Yoti.Auth/ActivityDetail.cs]: Technically, although no-one should be using this, it is a breaking change isn't it

Comment (echarrod) [reply src/Yoti.Auth/ActivityDetail.cs]: Yes you're right, I'll have to delay making these private until v3.0.0

File: src/Yoti.Auth/ActivityDetail.cs [hunk 2]
```diff
@@ -12,20 +12,42 @@ public enum ActivityOutcome { Success, ProfileNotFound, Failure, SharingFailure
     /// </summary>
     public class ActivityDetails
     {
+        public ActivityDetails(ActivityOutcome activityOutcome)
```
Comment (sidibos) [inline src/Yoti.Auth/ActivityDetail.cs]: Just curious - why do we need two constructors?

Comment (echarrod) [reply src/Yoti.Auth/ActivityDetail.cs]: If the share is successful we use one with 4 args, if it's not successful, we use the other one with just the outcome, this was just a way of making the object immutable after being constructed.

**Pull request merged**